website/public/design-system/colors_and_type.css

```
@@ -0,0 +1,361 @@
+   LightSpeedWP Agency — Colors & Type
+   Source: LightSpeedWP Design System.fig (Foundations / Tokens / Variables)
+   Supports light + dark modes; brand-blue first; technical, grounded.
+   ────────────────────────────────────────────────────────────────── */
+
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,400;0,500;0,600;0,700;1,400&family=Lora:ital,wght@0,400;0,500;1,400;1,500&display=swap');
+
+/* === FONTS =====================================================
+   Display + Body: locally-hosted brand variable fonts (uploaded).
+   Mono + Quote: Google Fonts (no licensed brand mono/serif on file). */
+@font-face {
+  font-family: "Inter";
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)